pkg/client/unversioned/daemon_sets_test.go

```diff
@@ -14,7 +14,12 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package unversioned
+package unversioned_test
```

Comment:
> btw, I believe your suggestion of "unversionedTest" will not compile,

Yea... I was wrong for the beginning. Did not take a very close look at the beginning.